packages/core/review-workflows/server/src/services/workflows.ts

```diff
@@ -87,6 +87,17 @@ export default ({ strapi }: { strapi: Core.Strapi }) => {
 
         createOpts = set('data.stages', mapIds(stages), createOpts);
 
+        if (opts.data.stageRequiredForPublish) {
+          const stageRequiredForPublish = stages.find(
+            (stage: any) => stage.name === opts.data.stageRequiredForPublish
+          );
+          if (!stageRequiredForPublish) {
+            throw new errors.ApplicationError('Stage required for publish does not exist');
```

Comment:
Grammatically it should be "stage required to publish" or "stage required for publication"